src/store/reducers/dform/initial/styles.json

```diff
@@ -3140,6 +3140,70 @@
             }
           ]
         },
+        {
+           "class": "block",
+           "label": "SEMEN ANALYSIS",
+           "children":[
+             {
+               "field": "Pathology.semen.analysis.collectionTime",
+               "label":"Collection Time",
+               "class":"textinput"
+
+             },
+             {
+              "field": "Pathology.semen.analysis.examinationTime",
+              "label":"Eximination Time",
+              "class":"textinput"
+             },
+             {
+              "field": "Pathology.semen.analysis.colour",
+              "label":"Colour",
+              "class":"textinput"
+             },
+             {
+              "field": "Pathology.semen.analysis.PH",
+              "label":"PH",
+              "class":"textinput"
+             },
+             {
+              "field": "Pathology.semen.analysis.volume",
+              "label":"Volume",
+              "class": "textunitinput",
+              "units": "ml"
```

Comment:
I think a definition like below is suitable for here:
```json
{
  ...
  "class": "textinput",
  "suffix": "ml"
}
```

`"class": "textunitinput"` is used for a field with multiple units like `height` which has units like `cm`, `m`, `ft`, `inch`, ... since `"class": "textunitinput"` provides the unit converting functionality.

In contrast, this field does not require multiple units and I think `textinput` is enough.
(The same discussion can be applied to the rest 😃